### ISSUE TITLE
INTERNAL ERROR report

### ISSUE THREAD

**Issue by MarcoGorelli** [psf/black#1429]
**Describe the bug** A clear and concise description of what the bug is.

I ran `black` on my codebase and got the following error:
```python-traceback
error: cannot format /mnt/c/data1/SSF_TV/pipeline/two_y_prediction.py: INTERNAL ERROR: Black produced code that is not equivalent to the source.  Please report a bug on https://github.com/psf/black/issues.  This diff might be helpful: /tmp/blk_pns90zrz.log  
```
As requested, I have opened an issue. Here is the log:
```diff
$ cat /tmp/blk_pns90zrz.log
--- src
+++ dst
@@ -2209,63 +2209,10 @@
                   type_comment=
                     None,  # NoneType
                   value=
                     Call(
                       args=
-                        Subscript(
-                          ctx=
-                            Load(
-                            )  # /Load
-                          slice=
-                            Slice(
-                              lower=
-                                None,  # NoneType
-                              step=
-                                None,  # NoneType
-                              upper=
-                                Name(
-                                  ctx=
-                                    Load(
-                                    )  # /Load
-                                  id=
-                                    'test_plan_month',  # str
-                                )  # /Name
-                            )  # /Slice
-                          value=
-                            Attribute(
-                              attr=
-                                'loc',  # str
-                              ctx=
-                                Load(
-                                )  # /Load
-                              value=
-                                Call(
-                                  args=
-                                    Constant(
-                                      value=
-                                        0,  # int
-                                    )  # /Constant
-                                  func=
-                                    Attribute(
-                                      attr=
-                                        'fillna',  # str
-                                      ctx=
-                                        Load(
-                                        )  # /Load
-                                      value=
-                                        Name(
-                                          ctx=
-                                            Load(
-                                            )  # /Load
-                                          id=
-                                            'ap2_data',  # str
-                                        )  # /Name
-                                    )  # /Attribute
-                                  keywords=
-                                )  # /Call
-                            )  # /Attribute
-                        )  # /Subscript
                       func=
                         Name(
                           ctx=
                             Load(
                             )  # /Load
@@ -6611,8 +6558,6 @@
     )  # /TypeIgnore
     TypeIgnore(
     )  # /TypeIgnore
     TypeIgnore(
     )  # /TypeIgnore
-    TypeIgnore(
-    )  # /TypeIgnore
 )  # /Module
```

**To Reproduce** Steps to reproduce the behavior:

1. Take this file '...'
2. Run _Black_ on it with these arguments '....'
3. See error

**Expected behavior** A clear and concise description of what you expected to happen.

**Environment (please complete the following information):**

- Version: [e.g. master]
- OS and Python version: [e.g. Linux/Python 3.7.4rc1]

**Does this bug also happen on master?** To answer this, you have two options:

1. Use the online formatter at https://black.now.sh/?version=master, which will use the
   latest master branch.
2. Or run _Black_ on your machine:
   - create a new virtualenv (make sure it's the same Python version);
   - clone this repository;
   - run `pip install -e .`;
   - make sure it's sane by running `python -m unittest`; and
   - run `black` like you did last time.

**Additional context** Add any other context about the problem here.
Am using `black, version 19.10b0 `.

As an aside, thanks for this truly great project, I'm very grateful to you all for your work on creating/maintaining `black`.

**Comment by ichard26** [comment]
Is it possible to get the source code that caused this error?

**Comment by tommilligan** [comment]
I believe this is a dupe of #1287, closed by #1315

**Comment by MarcoGorelli** [comment]
Hey all,

Sorry, this is something that happened at work a while ago, I thought the `.log` file would be enough - happy to close if it's been fixed
